Integrations/integration-cyberark_AIM.yml

```diff
@@ -97,8 +166,35 @@ script:
         return credentials;
     }
 
+    var resetCredentials = function(args) {
+        var result = sendRequestWithToken('PUT', credentialsUrl + '/' + args.accountId + '/ChangeCredentials', args, true);
+
+        return {
+            Type: entryTypes.note,
+            ContentsFormat: formats.json,
+            Contents: "Operation succeeded.",
+            ReadableContentsFormat: formats.markdown
+        };
+    };
+
+    var getAccountDetails = function(args) {
+        var result = sendRequestWithToken('GET', credentialsUrl, args, true);
+        if (!result) {
+            return 'no results found';
```

Comment:
1. Capital letter
2. add the account keywords. it might return after a while and this way it will be easier to understand what was searched INHO.